server/src/main/java/io/mantisrx/master/api/akka/route/Jackson.java

```diff
@@ -62,7 +67,9 @@
                     try {
                         return toJSON(mapper, null, u);
                     } catch (JsonProcessingException e) {
-                        throw new IllegalArgumentException("cannot marshall to Json " + u);
+                        String errMsg = "cannot marshal to Json " + u.toString().substring(0, 100);
```

Comment:
Do we need a check to make sure the string length >= 100?

Reply:
ah yes, good catch